tests/attr/neuron/test_neuron_conductance.py

```diff
@@ -155,12 +155,15 @@ def _conductance_input_sum_test_assert(
         ] = None,
     ):
         layer_cond = LayerConductance(model, target_layer)
-        attributions = layer_cond.attribute(
-            test_input,
-            baselines=test_baseline,
-            target=0,
-            n_steps=500,
-            method="gausslegendre",
+        attributions = cast(
```

Comment:
Is the cast necessary here ? I thought that the overrides for the attribute methods will solve the casting issue ?

Reply:
The issue is that this is for a layer method. The overrides help for regular and neuron attribution, but for layer, whether it returns a tuple is based on the layer input / output and not necessarily dependent on the input, thus it can't be represented clearly in type hints.

Reply:
I see, makes sense!